plutus-core/plutus-core.cabal

```diff
@@ -435,11 +436,13 @@ test-suite untyped-plutus-core-test
     type: exitcode-stdio-1.0
     main-is: Spec.hs
     hs-source-dirs: untyped-plutus-core/test
+    ghc-options: -O2 -threaded -rtsopts -with-rtsopts=-N
```

Comment:
It's funny we didn't have that already.

Reply:
I actually needed to add this to allow me to ensure that Hedgehog coverage didn't nondeterministically fail too often in a reasonable amount of time. But yes, it's strange that this isn't being used in _all_ test suites.